packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -151,6 +151,30 @@ export class Loan {
     return await this.client.call('setloantoken', [{ ...defaultData, ...loanToken }, utxos], 'number')
   }
 
+  /**
+   * Updates an existing loan token.
+   *
+   * @param {UpdateLoanToken} loanToken
+   * @param {string} loanToken.token Previous tokens's symbol, id or creation tx (unique)
+   * @param {string} loanToken.symbol New token's symbol (unique), no longer than 8
+   * @param {string} [loanToken.name] Token's name, no longer than 128
+   * @param {string} loanToken.priceFeedId Txid of oracle feeding the price
+   * @param {boolean} [loanToken.mintable = true] Token's 'Mintable' property
+   * @param {BigNumber} [loanToken.interest = 0] Interest rate
+   * @param {UTXO[]} [utxos = []] Specific UTXOs to spend
+   * @param {string} utxos.txid Transaction Id
+   * @param {number} utxos.vout Output number
+   * @return {Promise<string>} LoanTokenId, also the txn id for txn created to update loan token
+   */
+  async updateLoanToken (loanToken: UpdateLoanToken, utxos: UTXO[] = []): Promise<string> {
```

Comment:
if you can show the separation between old/current `token` parameter and the new properties of the Loan token, it would be better.
```suggestion
  async updateLoanToken (loanTokenIdentifier: string, newLoanTokenDetails: UpdateLoanTokenDetails, utxos: UTXO[] = []): Promise<string> {
```

Reply:
I shall replace with this
`async updateLoanToken (oldToken: string, updateLoanToken: UpdateLoanToken, utxos: UTXO[] = []): Promise<string>`

Reply:
I changed to this

`async updateLoanToken (oldToken: string, newTokenDetails: UpdateLoanToken, utxos: UTXO[] = []): Promise<string> {`